bin/probo-asset-receiver

```diff
@@ -1,33 +1,33 @@
 #! /usr/bin/env node
+/* eslint no-process-exit: 0, lines-around-comment: 0 */
+'use strict';
 
-var Loader = require('yaml-config-loader'),
-    bunyan = require('bunyan'),
-    yargs = require('yargs'),
-    loader = new Loader({ stopOnError: false }),
-    path = require('path')
-;
+var Loader = require('yaml-config-loader');
+var bunyan = require('bunyan');
+var yargs = require('yargs');
+var loader = new Loader({stopOnError: false});
+var path = require('path');
 
-var logger = bunyan.createLogger({name: "probo-asset-receiver"});
+var logger = bunyan.createLogger({name: 'probo-asset-receiver'});
 
 var app = require('..');
 
-var yargs = yargs
-  .describe('database-data-directory', 'The directory in which to store the LevelDB database.')
-  .alias('database-data-directory', 'D')
-  .describe('file-data-directory', 'The directory in which to store uploaded file assets.')
-  .alias('file-data-directory', 'F')
-  .describe('config', 'A path to a one or more configuration files or directories.')
-  .alias('config', 'c')
-  .describe('help', 'Display help message')
-  .alias('help', 'h')
-  .describe('host', 'The host to listen on (defaults to 0.0.0.0).')
-  .alias('host', 'H')
-  .describe('port', 'The port to listen on.')
-  .alias('port', 'p')
+var argv = yargs
+    .describe('database-data-directory', 'The directory in which to store the LevelDB database.')
+    .alias('database-data-directory', 'D')
+    .describe('file-data-directory', 'The directory in which to store uploaded file assets.')
+    .alias('file-data-directory', 'F')
+    .describe('config', 'A path to a one or more configuration files or directories.')
+    .alias('config', 'c')
+    .describe('help', 'Display help message')
+    .alias('help', 'h')
+    .describe('host', 'The host to listen on (defaults to 0.0.0.0).')
+    .alias('host', 'H')
+    .describe('port', 'The port to listen on.')
+    .alias('port', 'p')
+    .argv
 ;
 
-var argv = yargs.argv;
```

Comment:
I'm pretty sure this breaks the help config as configuration calls return a reconfigured `yargs` and do not modify the global. Then `.showHelp()` won't do the right thing.